ansible/all-in-one/roles/xsnippet/templates/xsnippet.yml.j2

```diff
@@ -0,0 +1,101 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: xsnippet
+spec:
+  containers:
+    - name: db
+      image: "{{ xsnippet_db_image }}"
+      env:
+        - name: POSTGRES_DB
+          value: xsnippet
+        - name: POSTGRES_USER
+          value: xsnippet
+        - name: POSTGRES_PASSWORD
+          value: xsnippet
+      volumeMounts:
+        - mountPath: /var/lib/postgresql/data
+          name: database-state
+    - name: api
+      image: "{{ xsnippet_api_image }}"
+      env:
+        - name: ROCKET_HOST
+          value: "{{ xsnippet_host|default('0.0.0.0') }}"
+        - name: ROCKET_PORT
+          value: "{{ xsnippet_port|default('8000') }}"
+        - name: ROCKET_DATABASE_URL
+          value: "postgresql://xsnippet:xsnippet@127.0.0.1:5432/xsnippet"
```

Comment:
nit: It'd be nice to see this in vars file because credentials might be different.

Reply:
Done.